pkg/util/ecs/metadata/clients.go

```diff
@@ -68,8 +70,18 @@ func V2() (*v2.Client, error) {
 	}
 
 	globalUtil.initV2.Do(func() {
-		globalUtil.v2 = v2.NewDefaultClient()
+		_ = globalUtil.initRetryV2.SetupRetrier(&retry.Config{
+			Name:              "ecsutil-meta-v2",
+			AttemptMethod:     initV2,
+			Strategy:          retry.Backoff,
+			InitialRetryDelay: 1 * time.Second,
```

Comment:
Maybe we should extract these 2 delays to constants given that we're using the same values for the 3 clients.

Reply:
done!